pkg/sdn/plugin/subnets.go

```diff
@@ -220,7 +221,15 @@ func (master *OsdnMaster) watchSubnets() {
 					log.Errorf("Error in deleting annotated subnet from master, name: %s, ip %s: %v", name, hostIP, err)
 					return nil
 				}
-				err = master.addNode(name, hostIP)
+				var hsAnnotations map[string]string
+				if vnid, ok := hs.Annotations[osapi.FixedVnidHost]; ok {
+					vnidInt, err := strconv.Atoi(vnid)
+					if err == nil {
+						hsAnnotations = make(map[string]string)
+						hsAnnotations[osapi.FixedVnidHost] = strconv.Itoa(vnidInt)
```

Comment:
There may be a choice to have the f5 for a particular project only. Then we will have the facility to change the value per deployment.